docs/data-sources/alert.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: "wavefront"
+page_title: "Wavefront: Alert"
+description: |-
+Get the information about a specific Wavefront alert.
+---
+
+# Data Source: wavefront_alert
+
+Use this data source to get information about a Wavefront alert by its ID.
+
+## Argument Reference
+* `id` - (Required) The ID associated with the alert data to be fetched.
+
+## Example Usage
+
+```hcl
+# Get the information about the alert.
+data "wavefront_alert" "example" {
+  id = "alert-id"
+}
+```
+
+## Attribute Reference
+
+* `name` - The name of the alert as it is displayed in Wavefront.
+* `id` - The ID of the derived metric in Wavefront.
+* `alert_type`- The type of alert in Wavefront.
+* `additional_information` - User-supplied additional explanatory information about this alert.
+* `target` - A email address or integration endpoint (such as PagerDuty or webhook) to notify when the alert status changes.
```

Comment:
An email address or integration endpoint (such as PagerDuty or webhook) to notify when the alert status changes.